### ISSUE TITLE
New release: touch events and virtualhostname mapping

### ISSUE THREAD

**Issue by miliuslisa** [jnschulze/flutter-webview-windows#171]
Hi! We are using this plugin in our work project and are eagerly waiting for the next release which includes pull request #163 and #164 (both of them already merged). Can you already tell when we can expect it? It would be a great help if we could use this in our project soon.

**Comment by venky9885** [comment]
@Doflatango  please prepare the documentation as requested by jnschulze  , Then he will merge PR and release new update in pub.dev It is  important requirement

**Comment by venky9885** [comment]
```dependency_overrides:
  flutter_inappwebview:
    git:
      url: https://github.com/jnschulze/flutter-webview-windows
      ref: master
```

current fix without publishing

**Comment by jnschulze** [comment]
I've just published [v0.2.1](https://pub.dev/packages/webview_windows/versions/0.2.1)
